src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -269,33 +266,61 @@ CHIP_ERROR Instance::Read(const ConcreteReadAttributePath & aPath, AttributeValu
     case Attributes::FeatureMap::Id:
         return aEncoder.Encode(mFeatureFlags);
 
-    case Attributes::SupportedWiFiBands::Id:
-        VerifyOrReturnError(mFeatureFlags.Has(Feature::kWiFiNetworkInterface), CHIP_NO_ERROR);
-        VerifyOrReturnError(mpDriver.Valid(), CHIP_NO_ERROR);
-
-        return aEncoder.EncodeList([this](const auto & encoder) -> CHIP_ERROR {
-            WiFiBand bandsBuffer[kMaxWiFiBands];
-            Span<WiFiBand> bands(bandsBuffer);
-            ReturnErrorOnFailure(mpDriver.Get<WiFiDriver *>()->GetSupportedWiFiBands(bands));
-
+    case Attributes::SupportedWiFiBands::Id: {
```

Comment:
Let's wait for https://github.com/project-chip/connectedhomeip/pull/31349 to land. Will fix this after.